FE/src/components/AddTaskModal/index.jsx

```diff
@@ -1,36 +1,28 @@
 import React, { useRef } from "react";
-import {
-  AddModal,
-  CloseButton,
-  TitleInput,
-  DateInput,
-  DescriptionInput,
-  ContainerInput,
-  AddDiffButton,
-  AddButtonsDiv,
-  AddButton,
-  Description,
-  ButtonsContainer,
-  DateTagdiv,
-  DescriptionContainer,
-  ModalContainer,
-} from "./style";
 import { AiOutlineClose } from "react-icons/ai";
 import { MdOutlineAddBox } from "react-icons/md";
+import {
+  AddButton, AddButtonsDiv, AddDiffButton, AddModal, ButtonsContainer, CloseButton, ContainerInput, DateInput, DateTagdiv, Description, DescriptionContainer, DescriptionInput, ModalContainer, TitleInput
+} from "./style";
 
 export default function AddTaskModal({
   closeHandler,
   addHandler,
   modalVisible,
   newTask,
+  dispatch,
 }) {
   const titleInput = useRef();
   const dateInput = useRef();
   const descriptionInput = useRef();
-  const addValue = () => {
-    newTask.title = titleInput.current.value;
-    newTask.description = descriptionInput.current.value;
-    newTask.date = dateInput.current.value;
+  const addValue = (type) => {
+    dispatch({
+      type: type, value: {
+        title: titleInput.current.value,
+        description: descriptionInput.current.value === "" ? "" : descriptionInput.current.value,
```

Comment:
???? why, deixar com ou sem é exatamente a mesma cena
```suggestion
        description: descriptionInput.current.value,
```

Reply:
Yah n faz mt sentido, vai ser sempre o current value